### ISSUE TITLE
Bufferbar scroll bar styling

### ISSUE THREAD

**Issue by Akin909** [onivim/oni#983]
This is a pretty subjective issue but the buffer bar once it overflows brings up a scroll bar beneath it which is fairly thick and takes up a bit of real estate and again subjectively is a little distracting, 
by setting the overlowX of that component to `overlay` and using.

```css
::-webkit-scrollbar {
    height: 3px /* as an example */
}
```
this would produce a more subtle effect better still I think might be having the bar only appear on hover over the area here's a screen shot of the suggested tweaks alongside, a similar implementation in vscode.

<img width="1245" alt="screen shot 2017-11-18 at 21 10 21" src="https://user-images.githubusercontent.com/22454918/32984846-07680124-cca6-11e7-9a88-0a2214e8eb73.png">
<img width="449" alt="screen shot 2017-11-18 at 21 11 21" src="https://user-images.githubusercontent.com/22454918/32984848-0d18b046-cca6-11e7-89ef-2fcba858ba15.png">

Definitely not wanting to overstep re. the look and feel of Oni, just thought i'd raise it and see what you thought.

**Comment by bryphe** [comment]
Just tried out your tweak, @Akin909 - looks great! Thanks for exploring this and opening the issue.

I thought that the thick scrollbar I was seeing was a problem with my laptop's rendering... but when I applied your changes it looked much better 😄 

Definitely think it makes sense to bring this in - are you up for opening a PR?

**Comment by Akin909** [comment]
Sure 👍

**Comment by bryphe** [comment]
Fixed by your PR #993 - thanks @Akin909 !
